Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificatesExtensionsTest.kt

```diff
@@ -147,12 +147,12 @@ class PersonCertificatesExtensionsTest : BaseTest() {
         certificates.remove(second)
         certificates.findHighestPriorityCertificate(time) shouldBe fourth
         certificates.remove(fourth)
-        certificates.findHighestPriorityCertificate(time) shouldBe third
-        certificates.remove(third)
         certificates.findHighestPriorityCertificate(time) shouldBe sixth
         certificates.remove(sixth)
         certificates.findHighestPriorityCertificate(time) shouldBe fifth
         certificates.remove(fifth)
+        certificates.findHighestPriorityCertificate(time) shouldBe third
+        certificates.remove(third)
```

Comment:
I wonder if this is expected ? now the third certificate is after the fifth ?!

Reply:
Because I've changed the order in the rules, now this order has been messed up, but this is expected. We could try to align the naming with the orders here but I wouldn't do it in this PR.

Reply:
Ok, that would be nice